eth/beacon/types/proposal_signed_data.py

```diff
@@ -0,0 +1,37 @@
+from eth_typing import (
+    Hash32,
+)
+import rlp
+
+from eth.rlp.sedes import (
+    uint64,
+    hash32,
+)
+
+
+class ProposalSignedData(rlp.Serializable):
+    """
+    Note: using RLP until we have standardized serialization format.
+    """
+    fields = [
+        # Fork version
+        ('fork_version', uint64),
```

Comment:
I think you can remove `fork_version` pending https://github.com/ethereum/eth2.0-specs/pull/169

```suggestion
```